src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/__init__.py

```diff
@@ -14,7 +14,8 @@ def __init__(self, cli_ctx=None):
         from azure.cli.core.commands import CliCommandType
         appservice_custom = CliCommandType(operations_tmpl='azure.cli.command_modules.appservice.custom#{}')
         super(AppserviceCommandsLoader, self).__init__(cli_ctx=cli_ctx,
-                                                       custom_command_type=appservice_custom)
+                                                       custom_command_type=appservice_custom,
+                                                       min_profile='2017-03-10-profile')
```

Comment:
The `2017-03-10-profile` is a bit magic (+1 day upon the current stack profile name). Would it make sense to be a bit explicit like `in_azure_stack=False`. My concern is if we update the stack profile name to be newer, then these command modules might show up again.

Reply:
The commands would show up again and that is a problem, I agree. However, `in_azure_stack` is, IMO, way too specific because the profile should determine whether a module appears or not--not whether it is in Stack. I would like to move toward making module loading explicit based on whether that module is in the profile or not. For this PR, I just found the handful of modules that were missing this annotation and added them. Most modules have it.